app/views/students/show.html.erb

```diff
@@ -1,5 +1,5 @@
-<h1><%= @student.name %></h1>
-<h2><%= @student.house %></h2>
+<h1><%= @students.name %></h1>
```

Comment:
A group of students does not have a `name`. An individual student, however, does have a `name`.